common/springUtilities/teamFunctions.lua

```diff
@@ -42,6 +43,7 @@ local function getSettings()
 					elseif luaAI:find("Scavengers") then
 						isScavengers = true
 						isAllyTeamValid = false
+						ScavTeamID = team
```

Comment:
Also set `scavAllyTeamID`

Reply:
if it automatically sets ally team to false if scavenger is found, what should I do to get scavAllyTeamID?

Reply:
`isAllyTeamValid` is just marking it as a non-player team, the value of the ally team ID is obtained from the loop on line 20, and the team ID value from the loop on line 27.

You'll need to replace both placeholder `_` with variables `allyTeamID` and `teamID` you can reference later.